src/main/electron-app/runnables/clean-up-deep-linking.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { beforeQuitOfBackEndInjectionToken } from "../../start-main-application/runnable-tokens/before-quit-of-back-end-injection-token";
+import lensProtocolRouterMainInjectable from "../../protocol-handler/lens-protocol-router-main/lens-protocol-router-main.injectable";
+
+const cleanUpDeepLinkingInjectable = getInjectable({
```

Comment:
why do you call it "deep linking"?

Reply:
Because the feature it tries to achieve is commonly known as deep linking. 

https://www.electronjs.org/docs/latest/tutorial/launch-app-from-url-in-another-app

ProtocolRouterMain etc. is technical, deep linking is behaviour we want to do. Naming should always describe to behaviour if possible.

Reply:
Okay